soccer/main.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Funny enough, main.py is the only place with the ``print`` statement. Most of the writing to things is ``click.secho``.